packages/transport/README.md

```diff
@@ -13,6 +13,15 @@ _You probably don't want to use this package directly._ For communication with T
 -   exposing single API for various transport methods:
     -   Trezor Bridge
     -   Webusb
+-   Create and expose typescript definitions based on protobuf definitions.
+
+## Updating messages
```

Comment:
maybe this should be mentioned is root README?

Reply:
good idea, but it should be done only after https://github.com/trezor/connect/pull/1077 is meged, because until then nothing changes.